recipes/libcbor/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibCborStackConan(ConanFile):
+    name = "libcbor"
+    license = "MIT"
+    homepage = "https://github.com/PJK/libcbor"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = """CBOR protocol implementation for C"""
+    topics = ("cbor", "serialization", "messaging")
+    exports_sources = ['CMakeLists.txt']
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake"
```

There aren't requirements, so nothing to consume using `find_package`